resources/application.ts

```diff
@@ -80,6 +80,14 @@ export class Applications extends BaseResource {
     public async verifyDocument(request: VerifyDocumentRequest): Promise<UnitResponse<ApplicationDocument>> {
         return this.httpPost<UnitResponse<ApplicationDocument>>(`/${request.applicationId}/documents/${request.documentId}/verify`, {data: request.data})
     }
+
+    public async download(request: DownloadDocumentRequest): Promise<Buffer> {
+        let path = `/${request.applicationId}/documents/${request.documentId}/download`
+        if (request.isBackSide)
+            path += "/back"
+
+        return this.httpGet(path, {responseType: "arraybuffer"})
+    }
```

Comment:
@axshani we need the download functions to be just like in checkDeposit and statements (provide an interface for sending responseEncoding and responseType for flexibility)